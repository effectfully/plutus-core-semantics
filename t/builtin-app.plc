[(lam x a x) (con 1 ! 128)]
