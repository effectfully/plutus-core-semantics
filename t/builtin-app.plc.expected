<generatedTop>
  <k>
    int ( 1 , 2 )
  </k>
  <env>
    .Map
  </env>
  <store>
    .Map
  </store>
</generatedTop>
